pkg/assembler/graphql/schema/certifyVEXStatement.graphql

```diff
@@ -79,22 +93,16 @@ input VexStatementInputSpec {
   collector: String!
 }
 
-"""
-PackageOrArtifactInput allows using PackageOrArtifact union as
-input type to be used in mutations.
-Exactly one of the value must be set to non-nil.
-"""
-input PackageOrArtifactInput {
-  package: PkgInputSpec
-  artifact: ArtifactInputSpec
-}
-
 extend type Query {
-  "Returns all CertifyVEXStatement"
+  "Returns all CertifyVEXStatement."
   CertifyVEXStatement(certifyVEXStatementSpec: CertifyVEXStatementSpec): [CertifyVEXStatement!]!
 }
 
 extend type Mutation {
-  "certify that an either a package or artifact has an associated VEX for a CVE, GHSA or OSV"
-  ingestVEXStatement(subject: PackageOrArtifactInput!, vulnerability: OsvCveOrGhsaInput!, vexStatement: VexStatementInputSpec!): CertifyVEXStatement!
+  """
+  Certify that a package or an artifact has an associated VEX for a vulnerability.
+
+  Note that setting `noVuln` in VulnerabilityInput is invalid for VEX statements!
+  """
+  ingestVEXStatement(subject: PackageOrArtifactInput!, vulnerability: VulnerabilityInput!, vexStatement: VexStatementInputSpec!): CertifyVEXStatement!
```

Comment:
why dont we keep this union type `OsvCveOrGhsaInput` so that it enforces that `noVuln` cannot be entered

Reply:
I renamed the struct first and then added the node. I can add back the other struct, but that would result in duplicated validation which is now hidden by the bool flag.

Reply:
Ah I see there is no client-side implementation to ingest via the `noVuln` node so this won't be an issue and you have a check on the query side to make sure it's not set for certifyVEX. Should be good then.